src/main/java/lekavar/lma/drinkbeer/blockentities/BeerBarrelBlockEntity.java

```diff
@@ -10,7 +10,7 @@
 import net.minecraft.nbt.CompoundTag;
 import net.minecraft.network.Connection;
 import net.minecraft.network.chat.Component;
-import net.minecraft.network.chat.TranslatableComponent;
+//import net.minecraft.network.chat.TranslatableComponent;
```

Comment:
Is there a reason to leave this commented here instead of removing it altogether?

Reply:
ah no - just forgot to remove it completly